openrpc.json

```diff
@@ -0,0 +1,1454 @@
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": [
+                "$$schema"
+              ],
+              "properties": {
+                "$$schema": {
+                  "type": "string"
+                },
+                "group_key": {
+                  "type": "string"
+                },
+                "group_value": {
+                  "type": "string"
+                }
+              }
+            }
+          },
+          "royalty": {
+            "type": "object",
+            "properties": {
+              "royalty_model": {
+                "type": "string",
+                "enum": [
+                  "creators",
+                  "fanout",
+                  "single"
+                ]
+              },
+              "target": {
+                "type": "string"
+              },
+              "percent": {
+                "type": "number"
+              },
+              "locked": {
+                "type": "boolean"
+              }
+            }
+          },
+          "creators": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "address": {
+                  "type": "string"
+                },
+                "share": {
+                  "type": "string"
+                },
+                "verified": {
+                  "type": "boolean"
+                }
+              }
+            }
+          },
+          "ownership": {
+            "type": "object",
+            "properties": {
+              "frozen": {
+                "type": "boolean"
+              },
+              "delegated": {
+                "type": "boolean"
+              },
+              "delegate": {
+                "type": "string"
+              },
+              "ownership_model": {
+                "type": "string",
+                "enum": [
+                  "Single",
+                  "Token"
+                ]
+              },
+              "address": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "AssetList": {
+        "type": "array",
+        "items": {
+          "$ref": "#/components/schemas/Asset"
+        }
+      },
+      "DisplayOptions": {
+        "type": "object",
+        "properties": {
+          "showZeroBalance": {
+            "type": "boolean",
+            "description": "Show accounts with zero balance",
+            "default": false
+          }
+        },
+        "required": [
+          "showZeroBalance"
+        ]
+      },
+      "GetByMethodsOptions": {
+        "type": "object",
+        "properties": {
+          "showUnverifiedCollections": {
+            "type": "boolean",
+            "description": "Show unverified collections"
+          },
+          "showGrandTotal": {
+            "type": "boolean",
+            "description": "Show grand total"
```

Comment:
Please figure out what this options do and add to the doc. You may ask teammates to figure this out or look at other refs.